airflow/models/__init__.py

```diff
@@ -23,6 +23,8 @@
 __all__ = [
     "DAG",
     "ID_LEN",
+    "Backfill",
+    "BackfillDagRun",
```

Comment:
There's an instruction not to add new models to this at line 22 above..

Reply:
removed